strongbox-web-core/pom.xml

```diff
@@ -50,7 +50,22 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-source-plugin</artifactId>
             </plugin>
-
+            <plugin>
```

Comment:
I think this plugin should still be kept in the `<profile/>`, or am I missing the point...?